templates/flutter/pubspec.yaml.twig

```diff
@@ -13,12 +13,16 @@ dependencies:
     sdk: flutter
   cookie_jar: ^3.0.1
   device_info_plus: ^4.1.2
-  flutter_web_auth_2: ^1.0.1
+  flutter_web_auth_2: ^1.1.1
   http: ^0.13.5
   package_info_plus: ^1.4.3+1
   path_provider: ^2.0.11
   web_socket_channel: ^2.2.0
 
+# TODO: Remove this when device_info_plus bumps their dependency of win32 to 3.0.0
+dependency_overrides:
```

Comment:
Since device_info_plus depends on win32 v 2.7.0 and flutter_web_auth_2 depends on win32 v 3.0.0, 
 you get version conflict error. And `flutter_web_auth_2`  latest version fixes windows implementation

Reply:
Doesn't need this anymore